.gitignore

```diff
@@ -121,3 +121,9 @@ dmypy.json
 .pyre/
 
 # End of https://www.gitignore.io/api/django
+
+# spreadsheet + data
+sample_data/
+
+# vscode
+.vscode/
```

Comment:
FYI @venkatabhishek I think a good practice is to put this in your global gitignore in your system user config file. It's fine to leave it here since so many of us use VS code, though.